### ISSUE TITLE
Remove PHP dependency

### ISSUE THREAD

**Issue by desiderantes** [flobrosch/valadoc-org#53]
Given that there's already some activity in web frameworks for Vala, like [Valum](https://github.com/valum-framework/valum), I think there's some value into using an existing Vala solution, support the underdog, etc.

**Comment by ZanderBrown** [comment]
definitely, a good point after all the raspberry pi website is (mostly) hosted on raspberry pi (https://www.raspberrypi.org/blog/the-little-computer-that-could/) why shouldn't valadocs be hosted on vala.

EDIT: I'm not going to pretend to know exactly how valadoc.org works but it appears to be mostly JS with some PHP doing database queries which should be doable in just about any language. I think this would probably depend more on a hosting provider supporting valum or the likes than anything else.

**Comment by btkostner** [comment]
I would think the first thing that needs to be done is get documentation generation that returns a non-markup file like xml or JSON or a database. As of right now, it's extremely hard to change anything because it requires changes to the way the generator works.

There has been a bit of work [here](https://github.com/eustasy/vala-docs.org/blob/continue-backend/doc-gen/JsonDoclet.vala) but it seems to have died down.
